src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -45,14 +48,15 @@ public override async Task InitializeAsync()
             if (_solutionName != null)
             {
                 VisualStudio.SolutionExplorer.CreateSolution(_solutionName);
-                VisualStudio.SolutionExplorer.AddProject(new ProjectUtils.Project(ProjectName), _projectTemplate, LanguageName);
+                VisualStudio.SolutionExplorer.AddProject(new ProjectUtils.Project(ProjectName), _projectTemplate, LanguageName, _templateParameters);
                 VisualStudio.SolutionExplorer.RestoreNuGetPackages(new ProjectUtils.Project(ProjectName));
 
                 // Winforms and XAML do not open text files on creation
                 // so these editor tasks will not work if that is the project template being used.
                 if (_projectTemplate != WellKnownProjectTemplates.WinFormsApplication &&
                     _projectTemplate != WellKnownProjectTemplates.WpfApplication &&
-                    _projectTemplate != WellKnownProjectTemplates.CSharpNetCoreClassLibrary)
+                    _projectTemplate != WellKnownProjectTemplates.CSharpNetCoreClassLibrary &&
+                    _projectTemplate != WellKnownProjectTemplates.BlazorApplication)
```

Comment:
Nit: Probably could update the comment above. Looks like it was already old before you got here, but now its older :)